source/Calamari/Kubernetes/Commands/KubernetesDeploymentCommandBase.cs

```diff
@@ -122,12 +122,18 @@ public override int Execute(string[] commandLineArguments)
             var conventionRunner = conventionProcessorFactory(runningDeploymentFactory(pathToPackage), conventions);
             try
             {
-                conventionRunner.RunConventions();
+                conventionRunner.RunConventions(logExceptions: false);
                 deploymentJournalWriter.AddJournalEntry(conventionRunner.Deployment, true, pathToPackage);
             }
-            catch (Exception)
+            catch (Exception e)
             {
                 deploymentJournalWriter.AddJournalEntry(conventionRunner.Deployment, false, pathToPackage);
+
+                if (e is KubernetesDeploymentFailedException || e is TimeoutException)
```

Comment:
These changes are to make the logs look neater and more user friendly. Basically, the conventions we are running handle known exceptions by logging the appropriate information and then throwing either a `KubernetesDeploymentFailedException` or a `TimeoutException`. We don't need to relog them again with a stack trace so instead we just just return -1.